generators/app/templates/_travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - "node" # Latest node version
```

Comment:
Indentation is off here. Been slowly trying to fix this, but it keeps spreading.

Reply:
And it would spread even more prolifically if it ended up in the project generator 😆